src/components/Form/Form.tsx

```diff
@@ -0,0 +1,53 @@
+import { Dispatch } from "react"
+import { FormActions } from "../../reducers/form-reducer"
+import { FormData } from "../../types"
+import CardHolderNameInput from "./components/CardHolderNameInput"
+import CardNumberInput from "./components/CardNumberInput"
+import ExpMonthInput from "./components/ExpMonthInput"
+import ExpYearInput from "./components/ExpYearInput"
+import CVCInput from "./components/CVCInput"
+
+
+type FormProps = {
+    state: FormData
+    dispatch: Dispatch<FormActions>
+}
+
+const Form = ({ state, dispatch }: FormProps) => {
+    return (
+        <section className="flex flex-col justify-center items-center mx-2 md:mt-14 lg:mt-0">
+            <div className="w-full max-w-sm">
+
+                <CardHolderNameInput state={state.cardholderName} dispatch={dispatch} />
+                <CardNumberInput state={state.cardNumber} dispatch={dispatch} />
+
+                <div className="flex flex-row justify-between gap-4 w-full">
```

Comment:
Detach these lines from 24 to 38 because eventhough they are part of the form it'd look cleaner to be in a component called Dates, inside you put such lines with a folder named components and inside ExpYearInput and ExpMonthInput. Like this:

Dates
|
|___components
                |_________ExpMonthInput
                |_________ExpYearInput